engine/cdn/item_logs_handler.go

```diff
@@ -120,3 +146,21 @@ func (s *Service) getSizeByProjectHandler() service.Handler {
 		return service.WriteJSON(w, size, http.StatusOK)
 	}
 }
+
+func (s *Service) checkItemLogsAuth(r *http.Request) (sdk.CDNAuthToken, error) {
+	vars := mux.Vars(r)
+	apiRef := vars["apiRef"]
+	tokenRaw := strings.TrimPrefix(r.Header.Get("Authorization"), "Bearer ")
```

Comment:
not sure about that the apiRef should be in URL and in the JWT, i guess the JWT only is mandatory

Reply:
Ok, Auth change will be in another PR.